### PR TITLE
Add Spotlights Winter Season 2021 newspost & season wiki article

### PR DESCRIPTION
**Pull request by venix12**
to-do before merging:
- [x] update badge images with winter ones
- [x] create feedback thread
- [x] create app cycle on spotlights web
- [x] open lobbies

### CODE REVIEW

File: news/2021-01-15-beatmap-spotlights-season-3-winter-2021.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+While we didn't change any essential things regarding league-aspect or general gameplay related things, we will ensure to have this season working smoother along with more recent badges and ranking updates, and better communication with the participants!
+
+We are also seeking new team members so make sure to check out the **[Seeking new curators](#applications)** section if you'd be interested in helping out!
+
+If you can't wait to try out new beatmaps though, **the Beatmap Spotlights lobbies are now open to play on the osu!lazer client**. If you don't know how to participate, check out the **[How to participate](#how-to-participate)** section!
+
+## <a id="how-to-participate"></a> How to participate in Beatmap Spotlights League?
+
+To participate in the Beatmap Spotlights League, you need to download the latest version of osu!lazer client from [GitHub releases page](https://github.com/ppy/osu/releases/latest). Once you're done installing, head on to multiplayer tab and find the right Spotlights lobbies, they can be recognized by a pink bar next to them.
```
Comment (Joehuu) [inline news/2021-01-15-beatmap-spotlights-season-3-winter-2021.md:19]:
```suggestion
To participate in the Beatmap Spotlights League, you need to download the latest version of osu!lazer client from [GitHub releases page](https://github.com/ppy/osu/releases/latest). Once you're done installing, head on to the multiplayer tab and find the right Spotlights lobbies, they can be recognized by a pink bar next to them.
```

File: news/2021-01-15-beatmap-spotlights-season-3-winter-2021.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+## <a id="applications"></a> Seeking new curators
+
+We are looking for for members that are both willing and capable of making decent beatmap selections for upcoming Beatmap Spotlights seasons.
```
Comment (Joehuu) [inline news/2021-01-15-beatmap-spotlights-season-3-winter-2021.md:23]:
```suggestion
We are looking for members that are both willing and capable of making decent beatmap selections for upcoming Beatmap Spotlights seasons.
```

File: news/2021-01-15-beatmap-spotlights-season-3-winter-2021.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+| Timeshift Lobby: Playlist A | Feb 19th - Feb 26th, 2021 |
+| Timeshift Lobby: Playlist B | Feb 26th - Mar 5th, 2021 |
+| Timeshift Lobby: Playlist C | Mar 5th - Mar 12th, 2021 |
+| Timeshift Lobby: Playlist D | Mar 12th - Mar 19th, 2021 |
+| Season Conclusion | Mar 19th, 2021 |
+
+You can check the finalized leaderboard for the previous season [here](https://docs.google.com/spreadsheets/d/e/2PACX-1vQ0_p56oQ2FcpYQoVKSZVhvFR7hX3tX7l0nUpCqlvTmBIEyDh0RdzjIg6WbjXTTtiA5-E4bd0aLxAV8/pubhtml
+).
+
+The Winter Season 2021 ledaerboard will be to be found on the [Beatmap Spotlights](/wiki/Beatmap_Spotlights) wiki article once first batch of lobbies conclude.
```
Comment (Joehuu) [inline news/2021-01-15-beatmap-spotlights-season-3-winter-2021.md:48]:
```suggestion
The Winter Season 2021 leaderboard will be to be found on the [Beatmap Spotlights](/wiki/Beatmap_Spotlights) wiki article once the first batch of lobbies conclude.
```

**Comment by Ephemeralis** [comment]
good to date this for tomorrow? (monday, 18-01-2021)

unexpected news schedule shift due to performance delaying FA release, don't really want to clip spotlights with another announcement as well

**Comment by venix12** [comment]
hmm, sounds alright, will reschedule lobbies and stuff

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**